plugins/backstage-plugin-github-insights/src/components/InsightsPage/InsightsPage.test.tsx

```diff
@@ -13,88 +13,69 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 import React from 'react';
-import { render, RenderResult, act, waitFor } from '@testing-library/react';
+import { act, render, RenderResult, waitFor } from '@testing-library/react';
 import { InsightsPage } from './InsightsPage';
 import { ThemeProvider } from '@material-ui/core';
 import { lightTheme } from '@backstage/theme';
 import { wrapInTestApp } from '@backstage/test-utils';
 import { EntityProvider } from '@backstage/plugin-catalog-react';
 
-import {
-  configApiRef,
-  githubAuthApiRef
-} from '@backstage/core-plugin-api';
+import { githubAuthApiRef } from '@backstage/core-plugin-api';
 
+import { ApiProvider, ApiRegistry, GithubAuth } from '@backstage/core-app-api';
+import { scmIntegrationsApiRef } from '@backstage/integration-react';
 import {
-  ApiProvider,
-  ApiRegistry,
-  GithubAuth,
-} from '@backstage/core-app-api';
-
+  createScmIntegrationsApiMock,
+  defaultIntegrationsConfig,
+} from '../../mocks/scmIntegrationsApiMock';
 
 const getSession = jest
   .fn()
   .mockResolvedValue({ providerInfo: { accessToken: 'access-token' } });
 
-const supportConfig = {
-  getString: (_2: string) => {
-    return '';
-  },
-
-  getOptionalString: (_3: string) => {
-    return null;
-  },
-
-  getConfigArray: (_: string) => {
-    return [];
-  },
-};
-
-const config = {
-  getOptionalConfigArray: (_: string) => [
-    { getOptionalString: (_2: string) => undefined },
-  ],
-
-  getOptionalConfig: (_: string) => {
-    return supportConfig;
-  },
-};
-
 const apis = ApiRegistry.from([
   [githubAuthApiRef, new GithubAuth({ getSession } as any)],
-  [configApiRef, config],
```

Comment:
The test setup changes as we no longer need to mock the config api but we do the scm api